### DISCUSSION TITLE
Stable releases frequency

### DISCUSSION THREAD

**Discussion by llfbandit** [Flutter-Bounty-Hunters/super_editor#1643] [Q&A] [answered]
Last super_editor release has been published 6 months ago.

I wish for more releases because:
- Packages are often outdated, and now with Flutter 3.16, this release is incompatible.
- For easier and faster maintenance when upgrading the package. While still in development, I think it should be more readable when getting smaller sets of breaking changes.
- For myself, I can't use dependency_overrides as I'm myself in a monorepo (transitive dependencies are not propagated to parent - using melos).

**Comment by llfbandit** [comment]
@matthew-carroll ?

**Comment by matthew-carroll** [comment, marked as answer]
Our goal is also to reach a more frequent release cadence. We've been blocked on the delivery of a major rework of the editing system. We need to implement undo/redo before we can cut the next release, but that's a big undertaking and hasn't been completed yet. After the next major release, we hope to release much more often.
